### ISSUE TITLE
System Error when running Ganache 1.2.1 on win32

### ISSUE THREAD

**Issue by wuzhanfly** [trufflesuite/ganache#863]
PLATFORM: win32
GANACHE VERSION: 1.2.1

EXCEPTION:
```
Error: listen EACCES 127.0.0.1:9545
    at Object.exports._errnoException (util.js:1024:11)
    at exports._exceptionWithHostPort (util.js:1047:20)
    at Server.setupListenHandle [as _listen2] (net.js:1302:19)
    at listenInCluster (net.js:1367:12)
    at doListen (net.js:1493:7)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+0ms: Starting server (version 1.2.1) with initial configuration: {"hostname":"127.0.0.1","port":9545,"network_id":5777,"default_balance_ether":100,"total_accounts":10,"unlocked_accounts":[],"locked":false,"vmErrorsOnRPCResponse":true,"verbose":false}
```

**Comment by seesemichaelj** [comment]
Thanks for reporting this! This issue is occurring because of a bug leaving the port open sometimes. The issue #665 was resolved in 1.2.x

### Please Update
We updated the interface and made it easier for you to change the settings when they don't work (like this issue).

Make sure all processes of Ganache are stopped (i.e. task manager or system monitor, etc.), and then head over to https://github.com/trufflesuite/ganache/releases/latest to get the most recent version of Ganache!

**Comment by seesemichaelj** [comment]
This issue is similar to that of #665. You likely have another program running on the same port (if this is not the case, and you're 100% there are no other programs or instances of ganache/ganache-cli, let me know).

I added a fix that should bring you to the settings page in https://github.com/trufflesuite/ganache/commit/d32d5f7919c45e07d52f2004e700f936d9aa91be and will be available in the next release.

In the meantime, you can find the config file for you to adjust the port at `%APPDATA%\Ganache\Settings`
